src/com/company/FlowerStore.java

```diff
@@ -0,0 +1,73 @@
+package com.company;
+
+import com.company.Flowers.Chamomile;
+import com.company.Flowers.Flower;
+import com.company.Flowers.Rose;
+import com.company.Flowers.Tulip;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class FlowerStore {
+
+    private int wallet = 0;
+
+    public FlowerStore() {
+    }
+
+    public int getWallet() {
+        return wallet;
+    }
+
+    public Flower[] sell(int rose, int chamomile, int tulip) {
+        Flower[] flow = new Flower[0];
+        final List<Flower> list = new ArrayList<>();
+        int counter = 0;
+        for (int i = 0; i < rose; i++) {
+            list.add(new Rose());
+            wallet += list.get(counter++).getPrice();
+
+        }
+        for (int i = 0; i < chamomile; i++) {
+            list.add(new Chamomile());
+            wallet += list.get(counter++).getPrice();
+        }
+        for (int i = 0; i < tulip; i++) {
+            list.add(new Tulip());
+            wallet += list.get(counter++).getPrice();
+        }
+        return list.toArray(flow);
+    }
+
+    public Flower[] sellSequence(int rose, int chamomile, int tulip) {
+        Flower[] flowers = new Flower[rose + chamomile + tulip];
+        int counter = 0;
+        int maxNumber = Stream.of(rose, chamomile, tulip).max((p1, p2) -> (p1.compareTo(p2))).get();
+        for (int i = 0; i <  maxNumber; i++) {
+            if (i < rose) {
+                flowers[counter] = new Rose();
+                wallet = flowers[counter++].getPrice();
```

Comment:
here you assign to wallet new value. check your wallet. now it returns wrong value